content/en/blog/_posts/0000-00-00-kube-proxy-non-privileged.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: blog
+title: "Non-privileged kube-proxy container"
+date: 0000-00-00
+slug: kube-proxy-non-privileged
+---
+
+**Author**: Lars Ekman
+
+This post describes how the `--init-only` flag added in K8s v1.29 can
+be used to perform configuration that requires privileged mode in an
+initContainer, while the main `kube-proxy` container may run with a
+stricter securityContext.
+
+Please note that `kube-proxy` can be installed in different ways, but
+installers that installs `kube-proxy` as a POD, like `kubeadm`, can
+now use this function for improved security.
+
+
+## Background
+
+It is undesirable to run a server container like `kube-proxy` in
+privileged mode. Security aware users wants to [use capabilities instead](
+https://github.com/kubernetes/kubernetes/issues/112171).
```

Comment:
Readers will expect this to take them to a guide, not an issue. I'd omit the link.